src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -333,13 +555,29 @@ export default {
 		font-weight: bold;
 		margin-top: rem-calc(20);
 	}
+}
+
+$card-width: rem-calc(290);
+$max-card-width: rem-calc(330);
+$card-margin: rem-calc(14);
+$card-half-space: rem-calc(14/2);
+
+.loan-card-group {
+	display: flex;
+	justify-content: center;
+}
+
+.cards-loan-card {
+	border-radius: 0.65rem;
+	box-shadow: 0 0.65rem $card-margin $card-half-space rgb(153, 153, 153, 0.1);
+	width: $card-width;
+	max-width: $max-card-width;
+	flex: 1 0 auto;
+	margin: $card-margin;
+}
```

Comment:
Initially used this for layout purposes but do see some anomalies on the card and will have to fill in some more styles later.